[Bug]: Firebase Hosting Deploy 用 CI がエラー

### 環境（Nx / Node / pnpm など）

Node v22.x

### 対象アプリ

web (Angular)

### ブラウザ / バージョン（web の場合）

_No response_

### OS

macOS

### 何が起きましたか？

PR を作成してマージしたら、エラーになった
https://github.com/gurezo/chirimen-device-dashboard/actions/runs/22519158467/job/65241358094

```
Running self-installer...
  Error: Multiple versions of pnpm specified:
    - version latest in the GitHub Action config with the key "version"
    - version pnpm@10.28.0 in the package.json with the key "packageManager"
  Remove one of these versions to avoid version mismatch errors like ERR_PNPM_BAD_PM_VERSION
      at readTarget (/home/runner/work/_actions/pnpm/action-setup/v4/dist/index.js:1:4977)
      at runSelfInstaller (/home/runner/work/_actions/pnpm/action-setup/v4/dist/index.js:1:4142)
      at async install (/home/runner/work/_actions/pnpm/action-setup/v4/dist/index.js:1:3154)
      at async main (/home/runner/work/_actions/pnpm/action-setup/v4/dist/index.js:1:445)
  Error: Error: Multiple versions of pnpm specified:
    - version latest in the GitHub Action config with the key "version"
    - version pnpm@10.28.0 in the package.json with the key "packageManager"
  Remove one of these versions to avoid version mismatch errors like ERR_PNPM_BAD_PM_VERSION
```

### 再現手順

PR を作成してマージ

### 期待される動作

Firebase Hosting にデプロイされる

### ログ / エラースタック

```shell

```

### チェックリスト

- [ ] README の Quick Start を確認しました

--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -19,8 +19,6 @@ jobs:
 
       - name: Setup pnpm
         uses: pnpm/action-setup@v4
-        with:
-          version: latest
 
       - name: Setup Node.js
         uses: actions/setup-node@v4

